.eslintrc.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
       },
     },
     {
-      files: ['@xen-orchestra/{web-core,lite,web}/**/*.{vue,ts}'],
+      files: ['@vates/**/*.{ts}', '@xen-orchestra/{web-core,lite,web}/**/*.{vue,ts}'],
```

Comment:
I don't think we should use Vuejs config for non-vuejs projects.

Reply:
This is used to tell eslint that there is TS in those directories. Without it, it treats the .ts files as JS and mistakes the typing for a syntax error.

Reply:
IMHO we should split the general TS config (which should match on all `*.{c,m,}ts`) files and the Vue.js config.

Reply:
It's an eslint config file for the whole repository, at the root of the repository, it's not a vuejs or TS config file.

Maybe we can remove this specific TS parameters and use .eslintrc.js files (that extends the root .eslintrc.js) in every package which use TS (parse-duration, web-core, lite, and web).

Reply:
I removed all the changes I made in .eslintrc.js to fix errors during test-lint. They did not reappear.